src/System.Private.CoreLib/shared/System/GCMemoryInfo.cs

```diff
@@ -17,7 +17,10 @@ namespace System
         public long MemoryLoadBytes { get; }
 
         /// <summary>
-        /// Total available memory for the GC to use when the last GC ocurred. By default this is the physical memory on the machine, but it may be customized by specifying a HardLimit.
+        /// If the environment variable COMPLUS_GCHeapHardLimit is set,
```

Comment:
```suggestion
        /// If the environment variable COMPlus_GCHeapHardLimit is set,
```

Unix env variables are case-sensitive